themes/bootstrap3/templates/Recommend/SideFacets.phtml

```diff
@@ -41,7 +41,7 @@
   <span class="sr-only"><?=$this->transEscAttr('page_reload_on_select_hint') ?></span>
 <?php endif; ?>
 <?php if ($hasVisibleCheckboxes): ?>
-  <div class="checkboxFilter">
+  <div class="checkbox-filters checkboxFilter">
```

Comment:
Following on from conversation on #4057, is there a compelling reason to keep the checkboxFilter class? Should we just replace all references to it with checkbox-filters for clarity? Since this will be in a major release, we could log it in the changelog if you're concerned about breaking local customizations. Or we could retain it in the markup but make sure we don't actually use it anywhere, and fully remove it in a future release. (Though it's too bad there's not really an easy way to deprecate and subsequently remove a CSS class!).

Reply:
I was just being (overly) cautious. It isn't used anywhere anymore, but I left it there just in case. Now removed. :) Mink tests still passing!